addons/mail/static/src/components/call_main_view/call_main_view.js

```diff
@@ -17,6 +17,16 @@ export class CallMainView extends Component {
         useComponentToModel({ fieldName: 'component' });
         useRefToModel({ fieldName: 'tileContainerRef', refName: 'tileContainer', });
         useUpdateToModel({ methodName: 'onComponentUpdate' });
+
+        onMounted(() => {
+            this.resizeObserver = new ResizeObserver(() => {
```

Comment:
no newline here (keep hooks in sequence, don't fluff code unnecessarily with empty newlines)